packages/closer/pages/stay/index.tsx

```
@@ -116,19 +116,25 @@ const StayPage = ({
+
             {`
             ${APP_NAME && APP_NAME === 'tdf' ? PLATFORM_NAME : ''}`}
           </Heading>
 
-          <div className='rich-text font-accent' dangerouslySetInnerHTML={{ __html: 
-              t.raw('stay_description')
-            }} />
-     
+          <div
+            className="rich-text font-accent"
+            dangerouslySetInnerHTML={{ __html: t.raw('stay_description') }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)